entities/PlotDriver.java

```diff
@@ -0,0 +1,27 @@
+package entities;
+
+import animation.ErrorMessageEvent;
+import rendering.AFrame;
+
+public abstract class PlotDriver extends Item {
+  AFrame frame;
+  
+  String name;
+  String description;
+  
+  int itemID;
+  
+  void use(PlayerCharacter player) {
+    new ErrorMessageEvent(this.frame, "You can't use that here.");
+  }
+  
+  void useInBattle(PlayerCharacter player, Opponent enemy) {
+    new ErrorMessageEvent(this.frame, "You can't use that here.");
```

Comment:
Not something you have to do, but (with a bit of commenting maybe) it might be shorter to just have this function call `use(player);`.